data/mods/MindOverMatter/monsters/feral_psychics.json

```
@@ -330,6 +330,224 @@
+      }
+    ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "HUMAN",
+      "CAN_OPEN_DOORS",
+      "PATH_AVOID_DANGER_1",
+      "PRIORITIZE_TARGETS",
+      "PHOTOKIN_MONSTER_IMMUNE"
+    ]
+  },
+  {
+    "id": "mon_feral_human_photokin2",
+    "type": "MONSTER",
+    "name": "feral beacon",
+    "description": "Almost blinding light radiates from this feral's skin. They seem to have greater control over their powers than most ferals, but they are still clearly insane.",
+    "default_faction": "zombie",
+    "looks_like": "chud",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "color": "yellow",
+    "symbol": "@",
+    "aggression": 30,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "bash", "amount": 4 } ],
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology" ],
+    "dodge": 4,
+    "harvest": "human",
+    "dissect": "dissect_human_sample_single",
+    "vision_day": 30,
+    "vision_night": 3,
+    "path_settings": { "max_dist": 30, "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true },
+    "death_drops": "feral_humans_death_drops_photo",
+    "zombify_into": "mon_zombie_survivor",
+    "anger_triggers": [ "FRIEND_DIED", "FRIEND_ATTACKED", "HURT" ],
+    "death_function": { "effect": { "id": "death_blind_oneinfour", "hit_self": true } },
+    "luminance": 50,
+    "upgrades": { "half_life": 45, "into": "mon_feral_human_photokin3" },
+    "special_attacks": [
+      [ "PARROT_AT_DANGER", 10 ],
+      {
+        "id": "psi_photokin2_lightdodge",
+        "type": "spell",
+        "spell_data": { "id": "photokinetic_light_dodge_monster", "min_level": 6 },
+        "cooldown": 10,
+        "monster_message": "%1$'s form is shifting with illusions!"
+      },
+      {
+        "id": "psi_photokin2_lightbeam",
+        "type": "spell",
+        "spell_data": { "id": "photokinetic_light_beam_monster", "min_level": 5 },
+        "cooldown": 15,
+        "monster_message": "%1$ fires a photon beam!"
+      },
+      {
+        "id": "psi_photokin2_lightflash",
+        "type": "spell",
+        "spell_data": { "id": "photokinetic_light_flash_monster", "min_level": 4 },
+        "cooldown": 15,
+        "monster_message": "%1$s concentrates and unleashes a super-luminous flash!"
+      }
+    ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "HUMAN",
+      "CAN_OPEN_DOORS",
+      "PATH_AVOID_DANGER_1",
+      "PRIORITIZE_TARGETS",
+      "PHOTOKIN_MONSTER_IMMUNE"
+    ]
+  },
+  {
+    "id": "mon_feral_human_photokin3",
+    "type": "MONSTER",
+    "name": "Feral Pulsar",
```

```suggestion
    "name": "feral pulsar",
```
Fits better with the way monsters are named, only a few are capitalized because they're unique or extremely rare.

---

Supposed to be on par with the other tier3 monsters of the mod which have unique names, so i renamed it to Atom Splitter.